pyproject.toml

```
@@ -243,6 +243,7 @@ doctest_subpackage_requires = [
     "astropy/table/mixins/dask.py = dask",
     "docs/* = numpy>=2",   # not NUMPY_LT_2_0 (PR 15065)
     "docs/units/type_hints.rst = python<=3.13",  # PYTHON_LT_3_14 (PR 18140)
+    "docs/units/structured_units.rst = pyerfa<2.0.1.7",  # PYERFA_LT_2.0.1.7
```

I think we use underscore, not period. Should be consistent for future grep to be useful. If you accept this, can probably skip CI. I will still be able to merge. Thanks!

```suggestion
    "docs/units/structured_units.rst = pyerfa<2.0.1.7",  # PYERFA_LT_2_0_1_7
```